cli/data.go

```diff
@@ -158,17 +158,22 @@ func getMatchingBinaryIDs(ctx context.Context, client datapb.DataServiceClient,
 		last = resp.GetLast()
 
 		for _, bd := range resp.GetData() {
-			ids <- bd.GetMetadata().GetId()
+			md := bd.GetMetadata()
+			ids <- &datapb.BinaryID{
```

Comment:
Added the switch to BinaryID here so that there's only a single RDK release needed, will remove in API/app in a separate PR.